runtime/permissions/lib.rs

```diff
@@ -476,6 +503,24 @@ impl PermissionState {
     info: impl Fn() -> Option<String>,
     prompt: bool,
   ) -> (Result<(), PermissionDeniedError>, bool, bool) {
+    if cfg!(unix)
+      && let Some(resp) = maybe_check_with_broker(name, &info)
```

Comment:
Nitpick: the `cfg!(unix)` could be dropped so that this logic is only in one place.